src/command_modules/azure-cli-container/azure/cli/command_modules/container/_params.py

```diff
@@ -0,0 +1,40 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+from azure.cli.core.commands import register_cli_argument
+from azure.cli.core.commands.parameters import enum_choice_list, location_type, name_type, resource_group_name_type
+from azure.cli.core.commands.validators import get_default_location_from_resource_group
+from azure.cli.core.util import CLIError
+
+# pylint: disable=line-too-long
+
+IP_ADDRESS_TYPES = ['Public']
+OS_TYPES = ['Windows', 'Linux']
+
+def environment_variables_format(value):
+    """Space separated values in 'key=value' format."""
+    try:
+        env_name, env_value = value.split('=')
```

Comment:
Does this work with environment variables with '=' in, e.g. database connection strings?

I tried the version in Cloud Shell and it didn't like my CosmosDB connection string and threw the error below.